carta/session.py

```diff
@@ -358,39 +359,58 @@ def set_starting_directory(self, path):
 
     # IMAGES
 
-    @validate(String(), String(r"\d*"), Boolean(), Constant(ArithmeticExpression))
-    def open_image(self, path, hdu="", complex=False, expression=ArithmeticExpression.AMPLITUDE):
-        """Open a new image, replacing any existing images.
+    @validate(String(), String(r"\d*"), Boolean())
+    def open_image(self, path, hdu="", append=False):
+        """Open or append a new image.
 
         Parameters
         ----------
         path : {0}
             The path to the image file, either relative to the session's current directory or an absolute path relative to the CARTA backend's root directory.
         hdu : {1}
             The HDU to select inside the file.
-        complex : {2}
-            Whether the image is complex. Set to ``False`` by default.
-        expression : {3}
-            Arithmetic expression to use if opening a complex-valued image. The default is :obj:`carta.constants.ArithmeticExpression.AMPLITUDE`.
+        append : {2}
+            Whether the image should be appended. Default is ``False``.
```

Comment:
This description should read:
```
Whether the image should be appended to existing images. By default this is ``False`` and any existing open images are closed.
```

(The same change should be applied to the other functions.)